src/lib/encoding.ts

```diff
@@ -305,12 +341,10 @@ export function parseManifestStream(stream: Readable): EventEmitter {
     ) {
       pathCount++;
       const p = keyPath[1];
+      paths[p] = data;
```

Comment:
@djwhitt what was the reason to empty the paths object before?
I have to keep it to handle fallback paths at the end of the stream.

Reply:
> I have to keep it to handle fallback paths at the end of the stream.

Main reason to clear it out is to conserve memory because manifests are unbounded in size. We should be able to continue discarding it as soon as we have seen both index and fallback paths I think.

Reply:
As `fallback` is limited to `id` the original `paths` behavior was kept.